doc/classes/PhysicsServer2D.xml

```diff
@@ -434,6 +434,34 @@
 				Creates a physics body.
 			</description>
 		</method>
+		<method name="body_enable_shape">
+			<return type="void">
+			</return>
+			<argument index="0" name="body" type="RID">
+			</argument>
+			<argument index="1" name="shape_idx" type="int">
+			</argument>
+			<argument index="2" name="enabled" type="bool" default="true">
+			</argument>
+			<description>
+				Sets whether or not the body's shape specified by [code]shape_idx[/code] is enabled.
+			</description>
+		</method>
+		<method name="body_enable_shape_one_way_collision">
```

Comment:
Same as `enable_shape_owner`, it would be more explicit with something like `body_set_shape_one_way_collision_enabled`, without the default value.